stl/inc/vector

```diff
@@ -3082,26 +3082,23 @@ public:
 
     template <class _Iter, enable_if_t<_Is_iterator_v<_Iter>, int> = 0>
     _CONSTEXPR20 iterator insert(const_iterator _Where, _Iter _First, _Iter _Last) {
-        difference_type _Off = _Where - begin();
-        _Insert(_Where, _First, _Last, _Iter_cat_t<_Iter>{});
-        return begin() + _Off;
-    }
+        const difference_type _Off_ret = _Where - begin();

```

Comment:
I would prefer if we could keep this as `_Off` and give the temporary in the fwd iterator branch a new name

Reply:
Srew that, I missed that we did decrement it below :(